packages/frontend/store/auth.store.ts

```diff
@@ -61,16 +46,17 @@ type Action = {
   changeChain: (chainId: ChainId) => void;
 };
 
-type AuthStore = State & Action;
+type AuthStore = AuthState & AuthActions;
 
-const initialState: InitialState = {
+const initialState: AuthState = {
   status: 'initial',
   address: undefined,
   ens: undefined,
   balance: undefined,
   chain: undefined,
   provider: undefined,
   walletName: undefined,
```

Comment:
no need to explicitly assign `undefined` values

Reply:
Yes, another quick refactor leftover